packages/wonder-blocks-dropdown/src/components/checkbox.tsx

```diff
@@ -18,50 +18,28 @@ type CheckProps = {
     disabled: boolean;
     /** Whether option item is selected. */
     selected: boolean;
-    /** Whether option item is pressed. */
-    pressed: boolean;
-    /** Whether option item is hovered. */
-    hovered: boolean;
-    /** Whether option item is focused. */
-    focused: boolean;
 };
 
 /**
  * The checkbox component used by OptionItem.
  */
 const Checkbox = function (props: CheckProps): React.ReactElement {
-    const {disabled, selected, pressed, hovered, focused} = props;
-    const activeBlue = mix(offBlack32, blue);
-    const clickInteraction = pressed || hovered || focused;
-
-    const bgColor = disabled
-        ? offWhite
-        : selected && !clickInteraction
-        ? blue
-        : white;
-    const checkColor = disabled
-        ? offBlack32
-        : clickInteraction
-        ? pressed
-            ? activeBlue
-            : blue
-        : white;
+    const {disabled, selected} = props;
 
```

Comment:
**note:** Similar to `check.tsx`, these states are now handled directly in `OptionItem` with css objects.